test/lib/ufe/testPixelMoveCmd.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2022 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+import mayaUtils
+import ufeUtils
+import fixturesUtils
+from testUtils import assertVectorAlmostEqual, assertVectorNotAlmostEqual
+
+import mayaUsd.lib
+
+from maya import cmds
+from maya import standalone
+
+import ufe
+
+@unittest.skipUnless(mayaUtils.ufeSupportFixLevel() > 0, "Requires pixelMove UFE support.")
```

Comment:
UFE support for pixelMove is implemented at fix level 1, so don't run if fix level is below that.